package-lock.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "pwa-kit",
-  "version": "2.7.0",
+  "version": "2.7.1-alpha.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "pwa-kit",
-      "version": "2.7.0",
+      "version": "2.8.0-dev",
```

Comment:
Something is wrong with the lock files, this shouldn't be `2.8.0-dev`.

Reply:
Before:
![Screenshot 2023-05-09 at 5 39 59 PM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/47404250/64406919-0da8-4311-8285-c7fd3d8eb9ac)

After:
![Screenshot 2023-05-09 at 5 56 31 PM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/47404250/0fab1d64-4283-4a20-af1a-b1583978cba5)

Thanks for catching that. I see it also. It's 2.7.1-alpha.0 at the top but 2.8.0-dev there.

I regenerated the lock files (and fixed an issue in ShopperLogin/mutation.test.js where an error is thrown if the project ends up building using commerce-sdk-isomorphic@1.10.1 vs commerce-sdk-isomorphic@1.9.0)

Reply:
This has been fixed.